lib/api/core/pluginsContext.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  ResponseObject: require('./models/responseObject'),
+  RealTimeResponseObject: require('./models/realTimeResponseObject'),
+
+  KuzzleError: require('./errors/kuzzleError'),
+  BadRequestError: require('./errors/badRequestError'),
+  ForbiddenError: require('./errors/forbiddenError'),
+  InternalError: require('./errors/internalError'),
+  NotFoundError: require('./errors/notFoundError'),
+  PartialError: require('./errors/partialError'),
+  ServiceUnavailableError: require('./errors/serviceUnavailableError'),
+  UnauthorizedError: require('./errors/unauthorizedError')
```

Comment:
As new errors get added regularly, it may be best to load them dynamically, by requiring all javascript files in the errors/ folder for instance. Or by using an index.js file.

What do you think?